Plant-for-the-Planet-org/planet-webapp
# manage subscription changes

- Manage subscription changes
- Edit Paypal Subscription implemented
- Pause donation even for trailing donations

Refer to the following spreadsheet to check the conditions:
https://docs.google.com/spreadsheets/d/1WkSoUfT2iv059Lee0YGnTCR8Q8tAUsvd9IfsQTGvslU/edit#gid=0

How to test:
1. Create some Donation Subscriptions from donate with plant-for-the-Planet (try paypal, sepa, sofort etc.
    for Paypal - https://donate-with-planet-git-feature-paypalsubscription-planetapp.vercel.app/
    for Sepa - https://donate-with-planet-git-feature-sepasubscription-planetapp.vercel.app/
    for others - paydev.pp.eco
2. Try editing, cancelling, pausing donations from the Webapp Dashboard
3. Report any unusual behaviour 

Note: Paypal will take some time to update the donation. Refresh the page 3-4 times, the changes should be updated.
